endpoints/getting-started/app.yaml

```diff
@@ -1,8 +1,6 @@
 runtime: go
 env: flex
 
-beta_settings:
-  # Enable Google Cloud Endpoints API management.
-  use_endpoints_api_management: true
-  # Specify the Open API specification.
-  endpoints_swagger_spec_file: openapi.yaml
```

Comment:
It is not referenced in app.yaml any more. Users would run "gcloud service-management deploy openapi.yaml" first, and then take the output of that command (service name and config_id) and update app.yaml only when they need to update their API Specification. This is consistent with the GCE/GKE model, and avoid re-deploying the Open API spec on every GAE Flex deployment.